### PR TITLE
Logging alerting config scripts

### PR DESCRIPTION
**Pull request by jamesgorrie**
## What's changing and why?
This allows us to programatically set the alerting settings we have on the logging cluster. This felt useful in the sense of we should keep what the settings are in code. Unfortunately there is no way currently of setting more of this in terraform.

I've gone for having it as a script that is run as and when, rather than something that is automatically called when you create the EC deployment with TF as we will be changing this throughout the lifecycle of the deployment and don't want to have to recreate every time there is change.

Currently the only thing we can do is add the slack connector to certain rules.

## `terraform plan` diff
No-op, just scripts

### CODE REVIEW

File: logging/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Logging
+
+## Scripts to manage the Elastic config
```
Comment (kenoir) [inline logging/README.md:3]: Would be nice to add:
- What is each one for specifically?
- How (and when) do we run them?

File: logging/kibana.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import fetch, { RequestInit } from "node-fetch";
+
+class KibanaClient {
```
Comment (kenoir) [inline logging/kibana.ts:3]: I'm surprised there isn't a Kibana client published by Elastic.

**Review by kenoir — APPROVED** [review]

File: logging/kibana.ts (hunk 2 above)
Comment (jamesgorrie) [reply logging/kibana.ts:3]: So was I - I'm not sure if it's about stability, or just another library to maintain...

File: logging/README.md (hunk 1 above)
Comment (jamesgorrie) [reply logging/README.md]: Nice spot, definitely a half formed doc. Updated - what do you think?

**Pull request merged**